tasks/main.yml

```diff
@@ -1,7 +1,8 @@
 ---
 - include: client.yml
   when: eks_clients_install
-
+  become: yes
```

Comment:
Please, leave default as `no`, in general the playbook should not have privilated access to install kube clients or its bin dependencies in your local env (laptop)